### ISSUE TITLE
Migration:refresh error

### ISSUE THREAD

**Issue by chiliec** [cybercog/laravel-love#120]
When migrations down appear these errors:
```bash
General error: 1828 Cannot drop column 'love_reactant_id': needed in a foreign key constraint 'posts_love_reactant_id_foreign' (SQL: alter table `posts` drop `love_reactant_id`)
General error: 1828 Cannot drop column 'love_reacter_id': needed in a foreign key constraint 'users_love_reacter_id_foreign' (SQL: alter table `users` drop `love_reacter_id`)
```
because need to also `dropForeign` before `dropColumn`.

**Comment by antonkomarev** [comment]
Good catch, @chiliec! Thank you.

Changes need to be done in these files:
- `src/Support/Database/Stubs/AddForeignColumn.stub`
- `src/Support/Database/Stubs/AddForeignNullableColumn.stub`

This is important, but not critical issue because this files are publishing to user application and could be changed. I have not much time to fix it right now. If you want - you could make a PR to push it forward.

**Comment by antonkomarev** [comment]
Fixed in [Love v8.1.2 release](https://github.com/cybercog/laravel-love/releases/tag/8.1.2)
